### ISSUE TITLE
Add an option to disable translating math

### ISSUE THREAD

**Issue by xuhdev** [mcmtroffaes/latexcodec#38]
Sometimes it is useful to disable the math translation: you may want the output string to be used for use in an HTML file, and maths are interpreted by MathJAX.

**Comment by mcmtroffaes** [comment]
Yes, shouldn't be too hard to implement. Do you know if there is a "standard" for passing extra configuration options to a codec?

**Comment by xuhdev** [comment]
I don't know... Maybe a name like `latex+nomath` is good?

**Comment by mcmtroffaes** [comment]
Sounds good to me.

**Comment by xuhdev** [comment]
Any progress :smiley: ?
